Gemfile

```diff
@@ -18,7 +20,7 @@ gem "rubocop", require: false
 # the one referred to in this update are no longer the same.
 # Hence by referring to it here we can lock the version to one we know allows
 # this project to run in our environments.
-gem "chromedriver-helper", "1.0.0"
+gem "chromedriver-helper", "2.1"
```

Comment:
You can actually drop this whole line and associated comment in the Gemfile. We needed to update chromedriver-helper due to the update to Chrome recently, but doing so means this will no longer run in our environments so the need to lock it has gone (and was never ideal anyway).